src/Sentry.Serilog/SentrySink.Structured.cs

```diff
@@ -0,0 +1,115 @@
+using Sentry.Internal.Extensions;
+using Serilog.Parsing;
+
+namespace Sentry.Serilog;
+
+internal sealed partial class SentrySink
+{
+    private static readonly SdkVersion Sdk = CreateSdkVersion();
+
+    private void CaptureStructuredLog(IHub hub, LogEvent logEvent, string formatted, string? template)
+    {
+        var traceHeader = hub.GetTraceHeader() ?? SentryTraceHeader.Empty;
```

Comment:
Is the `ParentSpanId` required in the protocol for StructuredLogging? An empty SpanId probably isn't that useful otherwise eh?

Reply:
No ... it's an optional "Default Attribute" ... the `trace_id` is _required_.

Oops ... I made this mistake in the Sentry-SDK and Microsoft.Extensions.Logging integrations, too.

I'll create a follow-up PR to fix this issue.